resolwe_bio/processes/workflows/chip_seq.yml

```diff
@@ -357,3 +357,341 @@
+            to fix-sized fragments. For example, if the size of binding region for your
+            transcription factor is 200 bp, and you want to bypass the model building by MACS,
+            this parameter can be set as 200. This option is only valid when --nomodel is set or
+            when MACS fails to build model and --fix-bimodal is on.
+        - name: shift
+          label: Shift
+          type: basic:integer
+          required: false
+          description: |
+            Note, this is NOT the legacy --shiftsize option which is replaced by --extsize! You
+            can set an arbitrary shift in bp here. Please Use discretion while setting it other
+            than default value (0). When --nomodel is set, MACS will use this value to move
+            cutting ends (5') then apply --extsize from 5' to 3' direction to extend them to
+            fragments. When this value is negative, ends will be moved toward 3'->5' direction,
+            otherwise 5'->3' direction. Recommended to keep it as default 0 for ChIP-Seq datasets,
+            or -1 * half of EXTSIZE together with --extsize option for detecting enriched cutting
+            loci such as certain DNAseI-Seq datasets. Note, you can't set values other than 0 if
+            format is BAMPE for paired-end data. Default is 0.
+        - name: band_width
+          label: Band width
+          type: basic:integer
+          required: false
+          description: |
+            The band width which is used to scan the genome ONLY for model building. You can set
+            this parameter as the sonication fragment size expected from wet experiment. The
+            previous side effect on the peak detection process has been removed. So this parameter
+            only affects the model building.
+        - name: nolambda
+          label: Use backgroud lambda as local lambda
+          type: basic:boolean
+          default: false
+          description: |
+            With this flag on, MACS will use the background lambda as local lambda. This means
+            MACS will not consider the local bias at peak candidate regions.
+        - name: fix_bimodal
+          label: Turn on the auto paired-peak model process
+          type: basic:boolean
+          default: false
+          description: |
+            Whether turn on the auto paired-peak model process. If it's set, when MACS failed
+            to build paired model, it will use the nomodel settings, the '--extsize' parameter
+            to extend each tags. If set, MACS will be terminated if paired-peak model is failed.
+        - name: nomodel
+          label: Bypass building the shifting model
+          type: basic:boolean
+          hidden: "tagalign"
+          default: false
+          description: |
+            While on, MACS will bypass building the shifting model.
+        - name: nomodel_prepeak
+          label: Bypass building the shifting model
```

Comment:
Same label and description as for `nomodel`. Is this right?

Reply:
Yes, the same situation as above.